contracts/rewards/ChildGaugeVoteRewards.sol

```diff
@@ -57,14 +54,11 @@ contract ChildGaugeVoteRewards is LzApp {
     constructor(
         address _aura,
         address _booster,
-        address _stashRewardDistro,
-        address _lzEndpoint
+        address _stashRewardDistro
```

Comment:
remove `_lzEndpoint` from constructor to keep same address accross all sidechains